templates/web/base/admin/triage/_list-filters.html

```diff
@@ -21,7 +21,7 @@
 [% END %]
 
         <p class="report-list-filters">
-            [% tprintf(loc('<label for="statuses">Show</label> %s reports <label for="filter_categories">about</label> %s', "The first %s is a dropdown of all/fixed/etc, the second is a dropdown of categories"), 'untriaged', mark_safe(select_category)) %]
+            [% tprintf(loc('<span>Show</span> %s reports <span>about</span> %s', "The first %s is a dropdown of all/fixed/etc, the second is a dropdown of categories"), 'untriaged', mark_safe(select_category)) %]
```

Comment:
The original HTML is correct for if JavaScript doesn't run, linking the label with the relevant multiple select. So I don't think the HTML should be changed as part of this.
If we don't want orphaned labels to exist, I think jquery-multi-select should be removing them/replacing with span, rather than simply removing the for. Or we could just do that in FMS code if we don't think that. @zarino ?

Reply:
This looks like it's been fixed as Matthew suggests

Reply:
@MorayMySoc sorry, it's because I didn't do a commit fixup if you check the files changed, the HTML is the same, the only change I made was to add id to the labels. Let me know if I misunderstood something.